### ISSUE TITLE
Dupe bug with Sync, OpenBlocks, and Tinker's construct

### ISSUE THREAD

**Issue by helium2s1** [MJRLegends/Space-Astronomy-Feedback-#169]
If you have a Tinker's Construct knapsack on your character and die with a clone in Sync storage, the knapsack and its contents will be placed in your gravestone, and will also be on your clone, resulting in a dupe.

Steps to reproduce:
1. create a sync clone and place it in storage
2. get a knapsack and place it on your character
3. put items in the knapsack
4. die; you should automatically transfer to your sync clone
5. harvest gravestone
6. profit

**Comment by MJRLegends** [comment]
I dont think there is anything i can do for this because it think this is a sync problem but the mod isn't updated anymore by the mod developer so probably wont be fixed but i will report to the OpenBlocks, and Tinker's construct mod developers to see if they can do something on there end for this.

**Comment by Blue64** [comment]
So it's a Must See Bug? lol that's as cheaty as it gets, one of the only fixes would be to discourage dying all together by absolutely spamming the skeleton spawning rate of the grave lol I would think OpenBlocks could handle this with a simple check for the clone in sync storage before adding the items to the grave.

**Comment by helium2s1** [comment]
As I've played with Sync more, I found you don't even have to die. just switching between multiple clones, especially across dimensions, will not only copy your tinker's knapsack and other gear on that tab, but will also copy galacticraft's wearable gear, and any soulbound armor you have on. I had one clone with nothing on and an empty inventory, that got a free flux-infused jetplate and dark steel armor, plus my oxygen and thermal setup, plus the knapsack with items when I switched to it.
